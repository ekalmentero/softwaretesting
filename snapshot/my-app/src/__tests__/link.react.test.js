import React from 'react';
import Link from '../Link.react';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Link page="http://www.ufrrj.bra">Site da UFRRJ</Link>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
