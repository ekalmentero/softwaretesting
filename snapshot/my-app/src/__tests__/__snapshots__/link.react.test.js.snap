// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="normal"
  href="http://www.ufrrj.br"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  Site da UFRRJ
</a>
`;
